ingestion-sink/src/test/java/com/mozilla/telemetry/ingestion/sink/io/PubsubReadIntegrationTest.java

```diff
@@ -12,22 +12,22 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicReference;
 import org.junit.Test;
 
 public class PubsubReadIntegrationTest extends TestWithPubsubResources {
 
+  private static final PubsubMessage TEST_MESSAGE = PubsubMessage.newBuilder()
+      .setData(ByteString.copyFrom("test".getBytes())).build();
+
   @Test
-  public void canReadOneMessage() throws Exception {
-    publisher
-        .publish(PubsubMessage.newBuilder().setData(ByteString.copyFrom("test".getBytes())).build())
-        .get(10, TimeUnit.SECONDS);
+  public void canReadOneMessage() {
+    publish(0, TEST_MESSAGE);
```

Comment:
Should we have a convenience method `publish(TEST_MESSAGE)` that assumes topic 0 for the simple case where the test only uses one topic?

Reply:
added for `TestWithSinglePubsubTopic`